content/post/072-js-ipfs-0.39.md

```diff
@@ -0,0 +1,140 @@
+---
+date: 2019-10-24
+url: 071-js-ipfs-0-39
+title: js-ipfs 0.39.0 released
+author: Alex Potsides
+---
+
+# 🔦 Highlights
+
+> Configure your node with preset profiles! Limit outgoing browser requests! Start your node from fresh with one command!
+
+```console
+$ npm install -g ipfs
+```
+
+## 🛠 Config profiles
+
+IPFS now supports the `jsipfs config profile` subcommand which you can use to update your configuration with certain presets suited for different use cases.
+
+You can examine available profiles with the `jsipfs config profile ls` command:
+
+```console
+$ jsipfs config profile ls
+server:
+ Recommended for nodes with public IPv4 address (servers, VPSes, etc.)...
+local-discovery:
+ Sets default values to fields affected by `server` profile...
+test:
+ Reduces external interference, useful for running ipfs in test environments...
+default-networking:
+ Restores default network settings. Inverse profile of the `test` profile.
+lowpower:
+ Reduces daemon overhead on the system. May affect node functionality...
+default-power:
+ Inverse of "lowpower" profile.
+ ```
+
+Apply them with `jsipfs config profile apply`:
+
+ ```console
+$ jsipfs config profile apply lowpower
+... output shows the difference between the old config and the new
+ ```
+
+You will need to restart your daemon for changes to take effect.
+
+Profiles can also be applied on init:
+
+```console
+$ jsipfs init --profile server
+```
+
+Or when starting the daemon (more on that in the 'Init and start' section below):
+
+```console
+$ jsipfs daemon --init-profile server
+```
+
+## 🌐 Concurrent HTTP requests limited in the browser
+
+Interacting with IPFS can cause a lot of preloading and other background requests to be made - most browsers can only have so many requests in flight to a single origin at once so this can cause requests to the same origins made outside of IPFS to take time as they wait their turn to be processed.
```

Comment:
Perhaps we could link to some additional context, for example:
```suggestion
Interacting with IPFS can cause a lot of preloading and other background requests to be made - [most browsers](https://bugs.chromium.org/p/chromium/issues/detail?id=285567) can only have so many requests in flight to a single origin at once so this can cause requests to the same origins made outside of IPFS to take time as they wait their turn to be processed.
```